src/diffusers/pipelines/pixart_alpha/pipeline_pixart_alpha.py

```diff
@@ -956,7 +956,11 @@ def __call__(
                     noise_pred = noise_pred
 
                 # compute previous image: x_t -> x_t-1
-                latents = self.scheduler.step(noise_pred, t, latents, **extra_step_kwargs, return_dict=False)[0]
+                if num_inference_steps == 1:
```

Comment:
can you provide a code example of how we would use one-step sampling with this pipeline? 
just try to understand so we can decide how to support it

Reply:
```python
from diffusers import PixArtAlphaPipeline, Transformer2DModel, DDPMScheduler

pipe = PixArtAlphaPipeline.from_pretrained(
        "PixArt-alpha/PixArt-XL-2-1024-MS",
        transformer=None,
        torch_dtype=weight_dtype,
    )
pipe.transformer = Transformer2DModel.from_pretrained("PixArt-alpha/PixArt-Alpha-DMD-XL-2-512x512", subfolder="transformer", torch_dtype=weight_dtype)
pipe.scheduler = DDPMScheduler.from_pretrained("PixArt-alpha/PixArt-Alpha-DMD-XL-2-512x512", subfolder="scheduler")

images = pipe(prompt='dog', timesteps=[400], num_inference_steps=1, output_type="pil",).images[0]
images.save('./dog.png')
```

Reply:
I think the `pred_prev_sample` and `pred_original_sample` should be same for last step though, here we only have one step so it should be the same?

i.e. `pred_original_sample_coeff` would be 1 and `current_sample_coeff` would be `0` 

https://github.com/huggingface/diffusers/blob/fc9fecc217e58d72c0d5796575c72088251ff4df/src/diffusers/schedulers/scheduling_ddpm.py#L475

Reply:
Ah, My current step is 400 and only use `step()` once, so I'm pretty sure that the `pred_original_sample_coeff` would not be 1.